tests/ui/or-patterns/exhaustiveness-unreachable-pattern.stderr

```diff
@@ -1,8 +1,10 @@
 error: unreachable pattern
   --> $DIR/exhaustiveness-unreachable-pattern.rs:8:9
    |
+LL |         (1 | 2,) => {}
+   |         -------- matches all the values already
 LL |         (1,) => {}
-   |         ^^^^
+   |         ^^^^ unreachable pattern
```

Comment:
@estebank do you have a better idea for the message here? I want to express that the pattern above already matches all the values matched by the pattern below.

Reply:
What about "unreachable pattern because it is already fully covered by a(n) earlier one(s)"?

Reply:
or slightly shorter: "unreachable pattern is already fully covered by an earlier one"

Reply:
I like this. What about the "matches all the values already"?